src/components/timeline/Track.css

```diff
@@ -23,26 +23,26 @@
 
   display: flex;
   flex-flow: row nowrap;
-
-  /* This padding is added to the button's padding as the perceived padding. For
-   * a global row, it will be replaced with a margin when it's selected. */
-  padding-left: var(--selected-left-border-width);
-  border-top: 1px solid var(--grey-30);
   background-color: #fff;
 }
 
 .timelineTrackRow.selected {
-  /* We replace the padding by some margin. Indeed the box-shadow used to draw
-   * the blue selected left border is drawn outside of the border box, so the
-   * 3px margin makes this space available. */
+  position: relative;
   padding-left: 0;
-  margin-left: var(--selected-left-border-width);
   background-color: #edf6ff;
+}
 
-  /* We use a box-shadow on the track row instead of using border, so
-   * that when we select adjacent tracks, we see just one line, without any
-   * border on top of it. */
-  box-shadow: calc(-1 * var(--selected-left-border-width)) 0 var(--blue-60);
+/* Showing a blue border on the left side of the track row to indicate that
+ * it's selected */
+.timelineTrackRow.selected::before {
+  position: absolute;
+  top: 0;
+  bottom: 0;
```

Comment:
I was gonna suggest to use "-1px" to cover the bottom border too, but this gives an issue with the _last_ track, where this increases the height when it's selected. So better leave it to 0 IMO.